Clean up HOOKS.md (#2585)
microsoft/BotFramework-WebChat: HOOKS.md

This function is only available in full bundle. The function will return the Adaptive Cards Host Config used for styling Adaptive Cards.

> To modify this value, change the props passed to Web Chat via style options.

Reviewer: I don't love this... Maybe:
```suggestion
To modify this value, change the value in the style options prop passed to Web Chat.
```

This function will return the image and initials of the user. Both image and initials are optional and can be falsy.

To set the avatar for the user, change the props passed to Web Chat via style options.

## `useConnectivityStatus`

```js
useConnectivityStatus(): [string]
```

> This function will return the connectivity status of Web Chat to Direct Line:

Reviewer: ```suggestion
This function will return the Direct Line connectivity status:
```

-  `connected`: Connected
-  `connectingslow`: Connecting is incomplete and more than 15 seconds have passed
-  `error`: Connection error
-  `notconnected`: Not connected, related to invalid credentials
-  `reconnected`: Reconnected after interruption
-  `reconnecting`: Reconnecting after interruption
-  `sagaerror`: Errors on JavaScript renderer; please see the browser's console
-  `uninitialized`: Initial connectivity state; never connected and not attempting to connect.

## `useDictateInterims`

```js
useDictateInterims(): [string[][]]
```

This function will return active interims processed from a dictation event.

> The first array represents separate sentences. The second array represents potential ambiguities or alternatives for the same sentence.

Reviewer: ```suggestion
The first array represents separate sentences while the second array represents potential ambiguities or alternatives for the same sentence.
```

## `useDictateState`

```js
useDictateState(): [string]
```

> This function will return a dictate state of one of the following:

Reviewer: ```suggestion
This function will return one of the following dictation states :
```

}) => React.Element
```

> This function is for rendering an activity inside a React element. The caller will need to pass `activity`, `timestampClassName`, and a render function for attachment. This function is a composition of `activityRendererMiddleware`, which is passed as a prop.

Reviewer: ```suggestion
This function is for rendering an activity inside a React element. The caller will need to pass `activity`, `timestampClassName`, and a render function for the attachment. This function is a composition of `activityRendererMiddleware`, which is passed as a prop.
```

The value is not the same as the props. Web Chat will merge the style options passed in props with default values specified in [`defaultStyleOptions.js`](https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/component/src/Styles/defaultStyleOptions.js).

> To modify the value of `styleOptions` state, change the props you pass to Web Chat via style options.

Reviewer: Lol

useTextBoxSubmit(): (setFocus: boolean) => void
```

This function will send the text box value as a message to the bot. In addition to the original `useSubmitSendBox` hook, this function will also scroll to bottom and, optionally, set focus to the send box.

> The focus is useful for a phone scenario, where the virtual keyboard will only be shown when a text box is focused.

Reviewer: ```suggestion
The focus is useful for a phone scenario where the virtual keyboard will only be shown when a text box is focused.
```
